spec/system/candidate_interface/adviser_sign_up/candidate_with_subject_match_selects_yes_on_adviser_interruption_spec.rb

```diff
@@ -0,0 +1,112 @@
+require 'rails_helper'
+
+RSpec.describe 'Candidate with a degree subject that matches an adviser teaching subject selects yes on adviser interruption', :js do
+  include CandidateHelper
+
+  it 'proceeds to the prefilled subject review step of the adviser sign up flow' do
+    given_i_am_signed_in_with_one_login
+    and_i_have_an_eligible_application # value of adviser_interruption_response is 'nil' by default
+    and_adviser_teaching_subjects_exist
+    and_rails_cache_is_enabled
+    and_analytics_is_enabled
+    and_enqueued_jobs_are_not_performed
+    and_the_adviser_sign_up_feature_flag_is_enabled
+    and_the_get_into_teaching_api_is_accepting_sign_ups
+    and_adviser_sign_up_jobs_can_be_enqueued
```

Comment:
I feel like this could be in a before block, as it is not really relevant to the substance of the test.